sympy/solvers/ode.py

```diff
@@ -1461,17 +1461,15 @@ def _test_term(coeff, order):
         return tuple(retlist)
 
 def separate_vars(f, symbols=[]):
-
-    from sympy import ordered
-
     if symbols:
         if not all((t.is_Atom for t in symbols)):
             raise ValueError("symbols must be Atoms.")
     elif symbols is None:
```

Comment:
this should probably be `elif symbols is []` because the user passed empty list (vs the user didn't pass anything and accepted the default).

Reply:
`elif symbols is []` won't work because the expression `[]` will always create a new list. That's why singleton objects like `None` are used for this kind of default. Mutable defaults sich as lists should be avoided in any case.

Why does this function even have optional arguments though?